src/lib/infiniteDiscovery/getInitialArtworksSample.ts

```diff
@@ -0,0 +1,17 @@
+import { opensearch } from "lib/apis/opensearch"
+
+export const getInitialArtworksSample = async (limit, artworksLoader) => {
```

Comment:
**suggestion:** We should probably add a constant for the limit here (`size=135`), as I believe the default size is 10. This way, we can ensure we get a random selection of 10 artworks from the entire collection.

Reply:
I don't think that passing size will randomise the results (but you probably mean that we can randomise them via javascript and pick N we need). I've implemented randomization using elasticsearch's function score